### DISCUSSION TITLE
Error when start search with OpenJDK without JFX

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#499] [General]
I received the error below and here is what happened and how I solved it.

![a6902d1a-d36f-41eb-9436-cb7e1f145579](https://user-images.githubusercontent.com/381594/116618590-a2764780-a915-11eb-8375-46a3442bbef6.jpeg)

After indexing a .dd image, I copied the result of the indexing folder (and also the .dd) to an external HD so that someone else could perform the searches on another computer, simply by placing the HD on the USB.
When running IPED-SearchApp.exe the screen above appeared, but this same HD worked perfectly on my computer.
After some checks I found that the computer was with an open-java. It was enough to install Java 8 from Oracle that everything worked well.

**Comment by lfcnassif** [comment]
Thanks @paulobreim, I think it is a good idea to warn users with OpenJDK installed without JavaFX (needed).

**Reply by lfcnassif** [reply]
With #621 the app will warn and exit. I started displaying "No JavaFX installed" msg in specific modules that need it, but as they are very important (map, html, eml, chat, metadata viewers), I think it is better to warn user and exit.

**Comment by Limaferreira** [comment]
Good night,

Taking advantage of the friend's post, I would like to know if it is possible, after indexing, to change the source location of the image generated from the HD. I'm performing the local indexing, through the D: drive of my computer, but I intend to make the content available over the network.

**Reply by lfcnassif** [reply]
Just open the analysis UI after moving your image and it will ask for the new path if not found, and will save it if the case folder is writeable.
